components/ui/cardHorizontal/index.tsx

```
@@ -22,19 +23,30 @@ const AdCard = dynamic(() => import("@components/ui/adCard"), {
+}) => {
   if (isLoading) return <CardHorizontalLoading />;
 
   if (event.isAd) {
-    return <AdCard event={event} />;
+    return <AdCard />;
   }
 
   return (
-    <CardContent event={event} isPriority={isPriority} isHorizontal={true} />
+    <CardContent
+      event={event as any}
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)